pinot-common/src/main/java/org/apache/pinot/common/utils/DataTable.java

```diff
@@ -46,8 +52,120 @@
   String RESIZE_TIME_MS_METADATA_KEY = "resizeTimeMs";
   String EXECUTION_THREAD_CPU_TIME_NS_METADATA_KEY = "executionThreadCpuTimeNs";
```

Comment:
This was technically never used in V2. It was only added in the previous PR but was never sent to the broker as part of DataTable metadata. So, can we remove it from here ?